docs/getting-started/windows-manual.md

```diff
@@ -87,6 +88,31 @@ Start-Service splunk-otel-collector
 
 The collector logs and errors can be viewed in the Windows Event Viewer.
 
+### Chocolatey installation
+
+#### Package Parameters
+The following package parameters are available:
+
+- `/SPLUNK_ACCESS_TOKEN`: The splunk access token (org token) used to send metric data to OTEL. If not specified default is to `1234`
```

Comment:
```suggestion
- `/SPLUNK_ACCESS_TOKEN`: The Splunk access token (org token) used to send metric data to Splunk Observability Suite.
```

Defaulting to an invalid token is a little unintuitive and sets the user up for issues out of the gate.  I think the Smart Agent doesn't start the service after installation if this is the case but I wonder if we should just make it required.

Reply:
I have made changes for making `SPLUNK_ACCESS_TOKEN` required to start service and removing default invalid value.